test_data/yam/agama_separate_boot.yaml

```diff
@@ -0,0 +1,23 @@
+---
+disks:
+  - name: vda
+    table_type: xfs
+    partitions:
+      - name: vda
+        fstype: xfs
+        mountpoints: [/home]
+        type: disk
+  - name: vdb
+    table_type: gpt
+    partitions:
+      - name: vdb1
+        fstype: btrfs
+        mountpoints: ["/var", "/usr/local", "/srv", "/root", "/opt", "/"]
+        type: part
```

Comment:
you don't need to add 'part' to any partition in this PR as you will be using the default later.

Reply:
removed.